webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -195,6 +219,7 @@ export default {
         teaserImage,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-12-11T16:00:34Z" title="Wednesday, December 11th 2019, 5:00:34 pm +01:00">Dec 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -177,6 +200,13 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      elem: null,
+      blurImage: false,
+    }
+  },
+  mounted() {
+    if (this.contribution && this.contribution.blurImage === true) {
+      this.blurImage = true
```


@ogerly please check how all the other form values are initiated – I know it's a bit confusing but I think we should stick to the existing behavior.

This would also mean we don't need the `mounted()` block anymore, which is always good, and we don't need to double the value for blurring images as `checkbox` and `blurImage` – it could just be one `blurImage` value that responds to the checkbox input!

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-12T08:46:59Z" title="Thursday, December 12th 2019, 9:46:59 am +01:00">Dec 12, 2019</time>_

---


mounted and put everything on blurImage. 
i hope i understood that correctly :)